src/utils/dom.js

```diff
@@ -591,6 +788,11 @@ export function indexOfTextNode(node, parent) {
 		return -1;
 	}
 	let nodeTextContent = node.textContent;
+	// Remove hyphenation if necessary.
+	if (nodeTextContent.substring(nodeTextContent.length - 1) == '-') {
```

Comment:
Is this from the hyphens we are adding when text is split in `hyphenateAtBreak()`?

If so we should check for `"\u2011"` and might need to follow up to remove the whatever character is set in `hyphenGlyph` option.

Reply:
I'll look at this and the next one - need a reminder of the context.

Reply:
Fixed, using both.